src/pages/ExcludeWordsStep2.js

```diff
@@ -0,0 +1,96 @@
+import { isSupportedBrowser } from "../utils/misc/browserDetection";
+
+import useExcludeInterest from "../hooks/useExcludeInterest";
+import useFormField from "../hooks/useFormField";
+
+import InfoPage from "./info_page_shared/InfoPage";
+import Header from "./info_page_shared/Header";
+import Heading from "./info_page_shared/Heading";
+import Main from "./info_page_shared/Main";
+import Form from "./info_page_shared/Form";
+import Button from "./info_page_shared/Button";
+import Footer from "./info_page_shared/Footer";
+import ButtonContainer from "./info_page_shared/ButtonContainer";
+import ArrowForwardRoundedIcon from "@mui/icons-material/ArrowForwardRounded";
+import AddRoundedIcon from "@mui/icons-material/AddRounded";
+import HighlightOffRoundedIcon from "@mui/icons-material/HighlightOffRounded";
+
+import InputField from "./info_page_shared/InputField";
+import Tag from "./info_page_shared/Tag";
+import TagContainer from "./info_page_shared/TagContainer";
+
+import redirect from "../utils/routing/routing";
+import strings from "../i18n/definitions";
+
+export default function ExcludeWordsStep2({ api, hasExtension }) {
+  const {
+    subscribedSearchFilters,
+    subscribeToSearchFilter,
+    removeSearchFilter,
+  } = useExcludeInterest(api);
+
+  const [excludedWords, handleExcludedWordsChange, resetExcludedWords] =
+    useFormField("");
+
+  function getLinkToNextPage() {
+    if (isSupportedBrowser() && hasExtension === false) {
+      return "/install_extension";
+    } else return "/articles";
+  }
+
+  function handleAddNewSearchFilter(e) {
+    e.preventDefault();
+    if (excludedWords) {
+      subscribeToSearchFilter(excludedWords);
+      resetExcludedWords();
+    }
+  }
+
+  return (
+    <InfoPage>
+      <Header>
+        <Heading>
+          Here you can add<br></br> unwanted words or&nbsp;phrases
+        </Heading>
+      </Header>
+      <Main>
+        <Form>
+          <InputField
+            value={excludedWords}
+            onChange={handleExcludedWordsChange}
+            helperText={strings.addUnwantedWordHelperText}
+            placeholder={strings.unwantedWordPlaceholder}
+          >
+            <Button className="small-square" onClick={handleAddNewSearchFilter}>
+              <AddRoundedIcon />
+            </Button>
+          </InputField>
+        </Form>
+        <TagContainer>
+          {subscribedSearchFilters.map((search) => (
+            <div key={search.id} searchremovabeid={search.id}>
+              <Tag
+                className={"outlined-blue square"}
```

Comment:
What is the square class? I couldn't find it in the inspector, and removing it didn't change anything?

Reply:
Good catch! This class no longer exists, so I removed it from `ExcludeWordsStep2.js` (I was experimenting with the radius of the filters at some point but decided to keep them round). There was also a CSS class selector for the button, placed next to the input field, called `small-square`, but I decided to rename it to a `small-square-btn`.